tests/integration_test/services_test.py

```
@@ -3886,6 +3886,172 @@ def test_upload_doc_content_file_type_validation_failure():
+        external_upload_info={"bsp": "360dialog"}
+    ).save()
+
+    response = client.get(
+        f"/api/bot/{pytest.bot}/channels/upload/media_upload",
+        headers={"Authorization": pytest.token_type + " " + pytest.access_token}
+    )
+
+    body = response.json()
+    UserMediaData.objects().delete()
+    Channels.objects().delete()
+    assert body["data"][0]['media_id'] == media_id
```

Ruff (0.12.2)

Use of `assert` detected

(S101)